spec/unit/downloader/cache_spec.rb

```diff
@@ -56,6 +56,11 @@ module Pod
       }
     end
 
+    it 'ensure path for spec uniq' do
```

Comment:
does this test cover what the fix is intended for? e.g. test that a pod with version 1.0.1 is saved properly compared to 1.0.podspec.json?

Reply:
yes. @request.slug currently is ok. and it ensure the path keep the @request.slug unchanged, which should be 1.0.1-#{checksum}, and end with .podspec.json extension. though the test is strongly couple with the currently implementation